README.md

```diff
@@ -14,29 +16,60 @@ A component, integrated with FT authentication and user data services, to add co
 - [Contact](#contact)
 - [Licence](#licence)
 
-### Markup
+## Markup
 
-Use the following markup to enable comments:
+These elements will be initialized automatically on the `o.DOMContentReady` event meaning you're unable to defer component initialisation.
+
+### Stream
+
+Use the following markup to enable a comment stream:
 
 ```html
-<div  class="o-comments" data-o-component="o-comments">
+<div class="o-comments"
+	id="comments"
+	data-o-component="o-comments"
+	data-o-comments-article-id="{article-id}">
+</div>
+```
+The comment stream is embedded in the element with a `comments` id attribute.
+
+### Count
+
+Add the following attribute to the markup to enable a comment count:
+
+```diff
+<div class="o-comments"
+-	id="comments"
+	data-o-component="o-comments"
++	data-o-comments-count
```

Comment:
Origami components can not have valueless attributes, I believe this is because they are hard to deal with as part of the JS DOM API.

Reply:
The part of the specification which states this is the fourth bullet point on this link --> https://origami.ft.com/spec/v1/markup/#markup-specification

Reply:
Got it 👍 Instead, I can set `data-o-comments-count` to `true` or `false`, what do you think?

Reply:
That's a good solution